.pre-commit-config.yaml

```diff
@@ -48,13 +48,20 @@ repos:
           - --strict
 
   # Shell script hooks
-  - repo: https://github.com/lovesegfault/beautysh
-    rev: v6.1.0
+  - repo: https://github.com/cisagov/pre-commit-shfmt
+    rev: v0.0.2
     hooks:
-      - id: beautysh
+      - id: shfmt
         args:
-          - --indent-size
+          # Indent by two space
```

Comment:
```suggestion
          # Indent by two spaces
```

Reply:
I rebased to fix this and it now lives in 406b688.